pocketimdb/movies/paginations.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+
+DEFAULT_PAGE = 1
+
+class CustomPageNumberPagination(PageNumberPagination):
+    page = DEFAULT_PAGE
+    page_size = 10
+
+    def get_paginated_response(self, data):
+        return Response({
+            'total': self.page.paginator.count,
+            'page': int(self.request.GET.get('page', DEFAULT_PAGE)),
```

Comment:
ovdje pricamo o response-u, tako da vec je obradjen page za koji treba da se vrate filmovi, tako da ne bi trebali da cupamo iz request-a nego iz onoga sto je vec izvrseno, iako ce ti brojevi biti isti. Tako da sto se tice trenutne stranice mozes na ovaj nacin vratiti `self.page.number`

Reply:
Da ovo mi je i nije imalo u potpunosti smisla, ali je u primeru sa linka bio neki komentar u tom delu koji me je zbunio i pomislio sam zbog njega da ipak mora ovako da se radi.. 
Ali da, ima smisla iz page property-ja to izvuci.